src/tbb/arena.cpp

```diff
@@ -526,22 +537,28 @@ void __TBB_EXPORTED_FUNC enqueue(d1::task& t, d1::task_group_context& ctx, d1::t
 void task_arena_impl::initialize(d1::task_arena_base& ta) {
     // Enforce global market initialization to properly initialize soft limit
     (void)governor::get_thread_data();
+    d1::constraints arena_constraints;
+
+#if __TBB_ARENA_BINDING
+    arena_constraints = d1::constraints{}
+        .set_core_type(ta.core_type())
+        .set_max_threads_per_core(ta.max_threads_per_core())
+        .set_numa_id(ta.my_numa_id);
+#endif /*__TBB_ARENA_BINDING*/
```

Comment:
```suggestion
#endif /*!__TBB_ARENA_BINDING*/
```

Reply:
As far as I can see, the `#endif` macro pertains to the `#if __TBB_ARENA_BINDING`. Therefore, this suggestion incorrectly hints at the corresponding beginning of the macro start.